src/store/store.js

```diff
@@ -0,0 +1,5 @@
+import { createStore, applyMiddleware } from "redux";
+import { authReducer } from "../reducers/authReducer";
+import { thunk } from "redux-thunk";
+
```

Comment:
IMO we can introduce `combineReducers` already as it will be needed later in the app, [doc](https://redux.js.org/usage/structuring-reducers/using-combinereducers)

Reply:
yes vaiya i introduce combineReducers on task-3 when i need  another reducer for add a blog post .
![image](https://github.com/mza-wtag/my-blog-react-app/assets/151821213/67e118b5-63a0-40bf-a130-a1419f975ecd)